pipelines/build/common/build_base_file.groovy

```diff
@@ -256,7 +262,30 @@ class Builder implements Serializable {
 
         return testList
     }
-
+    /*
+    Get the list of tests to dynamically run  parallel builds from the build configurations.
+    This function parses and applies this to the individual build config.
+    */
+    Map<String, ?> getDynamicParas(Map<String, ?> configuration, String variant) {
```

Comment:
getDynamicParams would be easier to read than getDynamicParas (but I won't be a stickler on it)